src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/eksportering/autoeksport/EksporteringServiceUtils.java

```diff
@@ -52,10 +52,21 @@ public class EksporteringServiceUtils {
         .collect(Collectors.toList());
   }
 
+  protected static SykefraværsstatistikkNæring hentSisteKvartalIBeregningenForSykefraværNæring(
+      List<SykefraværsstatistikkNæring> sykefraværNæring,
+      ÅrstallOgKvartal årstallOgKvartal) {
```

Comment:
Denne skjønte jeg ikke. Hvorfor sende inn `årstallOgKvartal` som argument, burde ikke informasjonen om hva som er siste årstall og kvartal i beregningen allerede finnes i listen over `SykefraværsstatistikkNæring`? 😕 

Enten trenger metoden et nytt navn, eller så må det forklares mer tydelig hva `årstallOgKvartal`-variabelen vi sender inn skal representere

Reply:
enig, denne funksjonen følger bare navnekonvensjonen til lignende funksjoner rundt, men de burde vel renames alle sammen. ser for så vidt at den ikke blir brukt, så sletter den